### PR TITLE
Auth pages and resume upload pages created.

### PR DESCRIPTION
**Pull request by sumit1804**
Issue Attached : #4 
Included :
1. Created login and signup pages.
2. Created Resume upload page. 
Note : Before running the client make sure to install all the packages `npm install`.

### CODE REVIEW

**Review by kanha638 — CHANGES_REQUESTED** [review]
@sumit1804 and @ChahitKolte2112 Make sure you add package-lock.json to `.gitignore` as it should not be pushed.

**Comment by kanha638** [comment]
> ready to merge this branch.

@sumit1804 
Don't add pull requests message like this its very unprofessional and irrelevant.

Please mention points of work that you did on this PR/change.

File: client/src/pages/ResumeUpload.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        console.log("removehandler");
+        setFile(null);
+        setTake(true);
+    }
+    return (
+        <Card sx={{ display: 'flex', gap: "20px", flexDirection: "column", justifyContent: 'center', alignItems: 'center', width: '98vw', height: '98vh', bgcolor: "#f6f9fc" }}>
+
+
+            <Card ref={wrapperRef}
+                sx={{ bgcolor: "white", width: "40%", height: "40vh", display: 'flex', justifyContent: "center", alignItems: "center", border: '2px dashed gray' }}
```
Comment (kanha638) [inline client/src/pages/ResumeUpload.tsx:57]: why for sx we don't have style object for styling ???

File: client/src/pages/loginPage.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+const ContainerStyle = {width:1,height:"98vh",display:"flex",flexDirection:{xs:"column",sm:"column",md:"row",lg:"row"}, alignItems:"center"}
+const ImageStyle = {width:{xs:1/4,sm:1/4, md:1, lg:1} , visibility:{xs:"hidden",sm:"hidden",md:"visible",lg:"visible"}}
+const LoginPage = () => {
+  return (
+    <Container sx={ContainerStyle}>
+        <Box sx={ImageStyle}>
+            <img src={logo} style={{width:"70%",margin:"15%"}}/>
+        </Box>
+        <Box width="100%"> 
+            <Box sx={{display:'flex',flexDirection:'column',gap:3,mx:{xs:2,sm:15,md:10,lg:6}}}>
```
Comment (kanha638) [inline client/src/pages/loginPage.tsx:14]: styles should be in different js object for code cleanliness

**Review by kanha638 — CHANGES_REQUESTED** [review]
please do above changes

File: client/src/pages/ResumeUpload.tsx (hunk 1 above)
Comment (sumit1804) [reply client/src/pages/ResumeUpload.tsx]: yes changes done

File: client/src/pages/loginPage.tsx (hunk 2 above)
Comment (sumit1804) [reply client/src/pages/loginPage.tsx]: yes done

**Review by kanha638 — APPROVED** [review]
Approved.

**Pull request merged**